server.js

```diff
@@ -0,0 +1,19 @@
+ var http = require('http');
+ console.log("starting");
+ var host = "127.0.0.1";
+ var port = 1337;
+
+ //create a server object:
+ var server = http.createServer(function(req, res) {
+ 	console.log("Recieved request:" + res.url);
+ 	res.writeHead(200, {
+ 		"Content-Type": "text/html"
+ 	});
+ 	console.log(req.method);
+ 	console.log(req.headers);
+ 	console.log(req.url);
+ 	res.end('Hello World');
```

Comment:
Code looks great, requirements said to have 

```
<html><body><h1>Hello World</h1></body></html>
```

returned from the server, not 'Hello World'

This also needs a MIME-type of 'text/html'